### DISCUSSION TITLE
Send Events to remote server?

### DISCUSSION THREAD

**Discussion by oguruma1218** [blakeblackshear/frigate#6370] [Ask A Question]
I have a third-party incident reporting tool. It has a REST API that can allow other services to send data to create entries in the database. Is there a way to send events from Frigate via REST API?

Rather than having the database server GET the events....

**Comment by NickM-27** [comment]
Not directly inside frigate but you could have a service / script read the MQTT events and send them wherever in whatever format you choose.
